charts/fluent-operator/values.yaml

```
@@ -180,6 +180,12 @@ fluentbit:
     systemd:
       enable: true
 
+  edge:
+    enable: false
+    prometheusRemoteWrite:
+      host: "<cloud-prometheus-service-host>"
+      port: "<cloud-prometheus-service-port>"
```

```suggestion
     ```suggestion
      # change the port to the port of a cloud side Prometheus-compatible server that can receive Prometheus remote write data
      port: "<cloud-prometheus-service-port>"
```